p2p/p2p-core/src/protocol/try_from.rs

```diff
@@ -1,149 +1,104 @@
 //! This module contains the implementations of [`TryFrom`] and [`From`] to convert between
 //! [`Message`], [`PeerRequest`] and [`PeerResponse`].
 
-use cuprate_wire::{Message, ProtocolMessage, RequestMessage, ResponseMessage};
+use cuprate_wire::{Message, ProtocolMessage};
 
-use super::{PeerRequest, PeerResponse};
+use super::*;
```

Comment:
I lied, `super::*` is only okay under the context it is in a `mod {}` in the same file (actually even then maybe not...)
```suggestion
use crate::{PeerRequest, PeerResponse, ProtocolRequest, ProtocolResponse};
```